epoch/api/chain_ws_api.md

```diff
@@ -411,32 +403,44 @@ Response:
 ```
 
 ## Data types
-### Header structure
+### Block structure
+A block can be either a key block or a micro block. [The key block](#key-block-structure) does not include any transactions. The micro block consists of a [header](#micro-block-header-structure) and a list of transactions.
+
+  | Name | Type | Description | Required |
+  | ---- | ---- | ----------- | -------- |
+  | transactions | [[transaction](#transaction-structure)] | List of transactions included in the block | No |
+
+**Note:** Genesis block lacks transactions and thus - `txs_hash`. Since it
+is the first block - it doesn't have `prev_hash`, too.
```

Comment:
", either"

Reply:
Not introduced by this PR - I'd fix it in separate PR.